doc/code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics/router.lua

```diff
@@ -0,0 +1 @@
+local vshard = require('vshard')
```

Comment:
What's about vshard bootstrap?

Reply:
Currently we describe `vshard.router.bootstrap()` in docs as an administration action rather than a part of a cluster application, for example:

https://www.tarantool.io/en/doc/latest/how-to/vshard_quick/#bootstrapping-a-cluster

cc @filonenko-mikhail